### DISCUSSION TITLE
Should I use the beta?

### DISCUSSION THREAD

**Discussion by huddy** [blakeblackshear/frigate#5248] [Ask A Question] [answered]
Hey

Firstly thanks for such a great project and to all the maintainers / contributors who make this project awesome :) 

I'm wondering if it's worth running the beta version? 

I had a quick read of the release notes but couldn't decide if it was worth it. 

What are biggest / best improvements that the community love? 

Also I know it's beta - and I don't expect it to be stable... but is it generally quite stable?

Cheers!

**Comment by NickM-27** [comment, marked as answer]
> What are biggest / best improvements that the community love?

Many users are very happy with the fact that live views are full FPS and have audio along with being much lower latency. The new system and storage page also make it much easier to understand how well your system is running and if more optimizations are needed. The ffmpeg presets also work to reduce CPU usage somewhat in many cases.
